ij-server/src/main/kotlin/org/jetbrains/research/ij/headless/server/SingleFileProjectConfigurator.kt

```diff
@@ -40,4 +36,13 @@ object SingleFileProjectConfigurator {
         logger.info("New file for language has been successfully created")
         return SingleFileProject(language, project, file.get(), disposable)
     }
+
+    enum class MainFile(val languageId: String, val fileName: String) {
+        KotlinMainFile(LanguageId.kotlin.name, "Main.kt"),
+        PythonMainFile(LanguageId.Python.name, "main.py");
+
+        companion object {
+            fun fromLanguage(language: Language) = MainFile.values().associateBy(MainFile::languageId)[language.id]
```

Comment:
I'd suggest to create the map only once `MainFile.values().associateBy(MainFile::languageId)` and then reusing it